src/command_modules/azure-cli-role/azure/cli/command_modules/role/_help.py

```diff
@@ -32,45 +33,39 @@
           short-summary: Space-separated list of scopes the service principal's role assignment applies to.
             Defaults to the root of the current subscription.
         - name: --role
-          short-summary: Role the service principal has in regard to resources.
+          short-summary: Role of the service principal.
     examples:
         - name: Create with a default role assignment.
           text: az ad sp create-for-rbac
         - name: Create using a custom name, and with a default assignment.
-          text: az ad sp create-for-rbac -n "http://MyApp"
+          text: az ad sp create-for-rbac -n "MyApp"
```

Comment:
I'm pretty sure this actually required the "http://" prefix.

Reply:
It actually doesn't. Semantic names for RBAC are very weird and the documentation has been unclear on them for a long time - it seems like their provided name does not actually require, or match on, the `http://` leader. So this is a valid example.

At some point I may need some more information on RBAC as its used in the CLI and how/if it directly relates to application authentication via a naming scheme.

Reply:
Yeah, as I understand the `http` is not required outside some specific (outdated?) scenarios and for automation should not be used.